lib/actions/populate.js

```diff
@@ -41,7 +41,7 @@ module.exports = function expand(route, options) {
             .modifyEager(relation, (builder) => {
 
                 builder.skipUndefined()
-                    .where(relation + '.' + keys.child.key, '=', keys.child.value)
+                    .where(Relation.tableName + '.' + keys.child.key, '=', keys.child.value)
```

Comment:
`relation` and `Relation` both floating around is confusing to me, but this is my first time peeking at Tandy so could be missing some context.

Reply:
I think this oughtta to be wrapped in [`ref()`](http://vincit.github.io/objection.js/#ref81) so that it can be escaped properly for the given SQL dialect.  Table names with uppercase letters need to be double-quoted in postgres if I recall correctly.